app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    get_command = command.split(" ")
+    if get_command[0] == "mv" and len(get_command) == 3:
+        command, old_file_name, new_file_name = get_command
+        directory = os.path.dirname(new_file_name)
+        if directory:
+            if not os.path.exists(os.path.dirname(new_file_name)):
+                os.makedirs(os.path.dirname(new_file_name), exist_ok=True)
```

Comment:
You already have the result of calculation in the `directory` variable, replace the following `os.path.dirname(new_file_name)` calls with `directory` variable.